iphone/Classes/TiUIWebView.m

```diff
@@ -518,6 +519,15 @@ - (void)setUrl_:(id)args
 	}
 }
 
+-(void)setBlacklistedURLs_:(id)args
+{
+    ENSURE_TYPE(args, NSArray);
+    if (blacklistedURLs) {
+        RELEASE_TO_NIL(blacklistedURLs);
+    }
+    blacklistedURLs = [[NSArray alloc] initWithArray:args];
```

Comment:
The release-cycle is clear, but think we don't work with ARC here, I see cases where the array could be GC'd while still being used. Especially when we have more requests over the time and the variable remains in idle.